src/poliastro/coordinates.py

```diff
@@ -11,72 +11,285 @@
 import astropy.units as u
 
 from poliastro.util import transform
+from poliastro.constants import J2000
 
 
-T = 0
-d = 0
-J2000 = time.Time('J2000', scale='tdb')
-
-Ja = 99.360714 + 4850.4046 * T
-Jb = 175.895369 + 1191.9605 * T
-Jc = 300.323162 + 262.5475 * T
-Jd = 114.012305 + 6070.2476 * T
-Je = 49.511251 + 64.3000 * T
-
-N = 357.85 + 52.316 * T
-
-bodies_orientations = {
-    "Mercury": {
-        "ra": 281.01 - 0.033 * T,
-        "dec": 61.45 - 0.005 * T,
-        "W": 329.548 + 6.1385025 * d
-    },
-    "Venus": {
-        "ra": 272.76,
-        "dec": 67.16,
-        "W": 160.20 - 1.4813688 * d
-    },
-    "Mars": {
-        "ra": 317.68143 - 0.1061 * T,
-        "dec": 52.88650 - 0.0609 * T,
-        "W": 176.630 + 350.89198226 * d
-    },
-    "Jupiter": {
-        "ra": 268.056595 - 0.006499 * T + 0.000117 * math.sin(Ja) + 0.000938 * math.sin(Jb) +
-        0.001432 * math.sin(Jc) + 0.000030 * math.sin(Jd) + 0.002150 * math.sin(Je),
-        "dec": 64.495303 + 0.002413 * T + 0.000050 * math.cos(Ja) + 0.000404 * math.cos(Jb) +
-        0.000617 * math.cos(Jc) - 0.000013 * math.cos(Jd) + 0.000926 * math.cos(Je),
-        "W": 284.95 + 870.5366420 * d
-    },
-    "Saturn": {
-        "ra": 40.589 - 0.036 * T,
-        "dec": 83.537 - 0.004 * T,
-        "W": 38.90 + 810.7939024 * d
-    },
-    "Uranus": {
-        "ra": 257.311,
-        "dec": -15.175,
-        "W": 203.81 - 501.1600928 * d
-    },
-    "Neptune": {
-        "ra": 299.36 + 0.70 * math.sin(N),
-        "dec": 43.46 - 0.51 * math.cos(N),
-        "W": 253.18 + 536.3128492 * d - 0.48 * math.sin(N)
-    },
-    "Pluto": {
-        "ra": 312.993,
-        "dec": 6.163,
-        "W": 237.305 - 56.3625225 * d
-    },
-    "Earth": {
-        "ra": 0.00 - 0.641 * T,
-        "dec": 90.00 - 0.557 * T,
-        "W": 190.147 + 360.9856235 * d
-    }
-}
-
-
-def body_centered_to_icrs(r, v, source_body, epoch=None, rotate_meridian=False):
+def mercury_rot_elements(epoch=J2000):
+    """Provides Mercury rotational elements given an epoch.
+
+    Provides north pole of body and angle to prime meridian
+
+    Parameters
+    ----------
+    epoch : ~astropy.time.Time, optional
+        Epoch, default to J2000.
+
+    Returns
+    -------
+    ra, dec, W: tuple (~astropy.units.Quantity)
+        Right ascension and declination of north pole, and angle of the prime meridian.
+
+    """
+
+    T = (epoch.tdb - J2000).to('day').value / 36525
+    d = (epoch.tdb - J2000).to('day').value
+
+    ra = (281.01 - 0.033 * T) * u.deg
```

Comment:
@anhiga and I already clarified in private chat that, although putting these constants inside the `Body` definitions, things get hairy for Jupiter, the Moon and other bodies... So they are fine here.